tests/language/completion/test_mysql_naive_completion.py

```diff
@@ -0,0 +1,109 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
```

Comment:
sort these import statements alphabetically

Reply:
It already seem to be in the correct order? I believe the standard is to separate library imports and local imports.